### DISCUSSION TITLE
<title>How to delete a file

### DISCUSSION THREAD

**Discussion by LanguageDescriptions** [community/community#81410] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have about 20 files that I want to delete from a repository. At the moment, I can only access them through the web interface, not the desktop program. What I'm looking for but can't find no the interface is a file-select checkbox then once I've selected the 20 doomed files, drop them in the trash. Problem is I can't find such an option. Where is it, or how do I find it? Firefox (Windows) in case that matters.

Thanks, d.o

**Comment by faridnec** [comment]
hey, If you have Git installed on your Windows machine:
1. Clone the repository locally.
2. Use the `git rm` command to delete multiple files at once: `git rm <file1> <file2> <file3> ...`
3. Commit and push the changes: `git commit -m "Delete multiple files"` `git push`

But remember that deleting files through the web interface permanently removes them from the repository's history and if you need to revert the deletion later, you'll have to rely on backups or older branches.

**Comment by LanguageDescriptions** [comment]
Right, if I could I would, but the desktop is not an option for me. (The problem is that I can't add a second repository and if I try I have to uninstall everything and start over: there can be only one. I suspect it's my computer and this will go away when I replace it). That's why I'm looking for a browser-based deletion.

**Reply by faridnec** [reply]
If desktop options are limited for you, browser-based deletion on GitHub is file by file. You'll need to delete them individually.
